universal/ubi8/Dockerfile

```
@@ -371,7 +371,7 @@ RUN <<EOF
 set -euf -o pipefail
 TEMP_DIR="$(mktemp -d)"
 cd "${TEMP_DIR}"
-KN_VERSION="0.26.0"
+KN_VERSION="1.13.0"
 KN_ARCH="linux-amd64"
 KN_BIN="kn-${KN_ARCH}"
 KN_BIN_URL="https://github.com/knative/client/releases/download/v${KN_VERSION}/${KN_BIN}"
```

It seems the link was updated 
```suggestion
KN_BIN_URL="https://github.com/knative/client/releases/download/knative-v${KN_VERSION}/${KN_BIN}"
```